app/views/release_notes/_v5.0.2.1.html.erb

```diff
@@ -0,0 +1,17 @@
+ <h3>AACT 5.0.2.1<i>(October 28, 2021)</i></h3>
+ 
+ <h5>Release notes - AACT - Version aact-core-5.0.2</h5>
```

Comment:
This can remove this line
`<h5>Release notes - AACT - Version aact-core-5.0.2</h5>`
 since the version is given above